projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-details/quote-details.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Cart, RoutingService, UserIdService } from '@spartacus/core';
+import { FSTranslationService } from './../../../../core/i18n/facade/translation.service';
+import { QuoteService } from '../../../../core/my-account/facade/quote.service';
+import { FSCartService } from './../../../../core/cart/facade/cart.service';
+import { combineLatest, Observable, Subscription } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-fs-quote-details',
+  templateUrl: './quote-details.component.html',
+})
+export class QuoteDetailsComponent implements OnInit, OnDestroy {
+  quote$: Observable<any>;
```

Comment:
Can we maybe expand `InsuranceQuote` interface to start getting types? And maybe merge 2 `InsuranceQuote` interfaces (one declared in occ.models.ts and other in quote.model.ts) into one.

Reply:
So it could be done, but maybe we can discuss it more in details, because as I saw both are used, also I'm not sure if those (price, category) attributes are same for those interfaces